cfme/tests/ansible/test_embedded_ansible_crud.py

```diff
@@ -82,3 +86,49 @@ def is_ansible_pod_stopped(enabled_embedded_appliance):
         assert wait_for(is_nginx_stopped, func_args=[enabled_embedded_appliance], num_sec=30)
     else:
         assert wait_for(is_ansible_pod_stopped, func_args=[enabled_embedded_appliance], num_sec=180)
+
+
+@pytest.mark.tier(1)
+def test_embed_tower_event_catcher_process(appliance, enabled_embedded_appliance):
+    """
+    EventCatcher process is started after Ansible role is enabled (rails
+    evm:status)
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: critical
+        initialEstimate: 1/4h
+        tags: ansible_embed
+    """
+    result = appliance.ssh_client.run_command(
+        "cd /var/www/miq/vmdb; rake evm:status | grep 'EmbeddedAnsible'"
```

Comment:
Required:
you can use direct rake command  like
```
result = appliance.ssh_client.run_rake_command("evm:status | grep EmbeddedAnsible").output
```

Reply:
I have updated with above suggestion.